.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -26,7 +26,7 @@ https://github.com/testing-library/testing-library-docs
 
 ### Relevant code or config:
 
-```js
+```javascript
```

Comment:
js is fine/preferred (we have to use `jsx` a lot, too) but we can leave this change in